lib/puppet-strings/hiera/data.rb

```diff
@@ -64,6 +78,31 @@ def to_s
 
     private
 
+    def value_to_s(value)
```

Comment:
i don't think this should be required but was needed for a clean diff of 

https://github.com/b4ldr/puppet-unbound/pull/1/files

Reply:
yes i completely agree although im not sure where the best place is

Reply:
this got removed in favour f using inspect although the genral point of moving this display logic is still valid but ill resolve this for now